tests/util/test_logger.py

```diff
@@ -1,8 +1,19 @@
+import os
+import pytest
+
 from fidesops.util import logger
 from fidesops.util.logger import NotPii, MASKED
 
 
-def test_logger_masks_pii() -> None:
+@pytest.fixture(scope="function")
+def toggle_testing_envvar() -> None:
```

Comment:
This is useful to explicitly check areas where we use the `os.environ["TESTING"]` in production code